CRITICAL - 2023-03-24 02:20:14 --> Undefined index: nama_anggota
in APPPATH\Controllers\Auth.php on line 203.
 1 APPPATH\Controllers\Auth.php(203): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index: nama_anggota', 'APPPATH\\Controllers\\Auth.php', 203, [...])
 2 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Auth->CekLoginAnggota()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2023-03-24 02:52:57 --> mysqli_sql_exception: Table 'kelas' already exists in C:\projectci4\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\projectci4\system\Database\MySQLi\Connection.php(292): mysqli->query('CREATE TABLE `k...', 0)
#1 C:\projectci4\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `k...')
#2 C:\projectci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `k...')
#3 C:\projectci4\system\Database\Forge.php(509): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `k...')
#4 C:\projectci4\app\Database\Migrations\2023-03-24-073947_Web.php(42): CodeIgniter\Database\Forge->createTable('kelas')
#5 C:\projectci4\system\Database\MigrationRunner.php(868): App\Database\Migrations\Web->up()
#6 C:\projectci4\system\Database\MigrationRunner.php(187): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\projectci4\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\projectci4\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\projectci4\system\CLI\CommandRunner.php(65): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\projectci4\system\CLI\CommandRunner.php(51): CodeIgniter\CLI\CommandRunner->index(Array)
#11 C:\projectci4\system\CodeIgniter.php(897): CodeIgniter\CLI\CommandRunner->_remap('index', Array)
#12 C:\projectci4\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#13 C:\projectci4\system\CodeIgniter.php(336): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\projectci4\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#15 C:\projectci4\spark(98): CodeIgniter\CLI\Console->run()
#16 {main}
ERROR - 2023-03-24 03:20:55 --> mysqli_sql_exception: Unknown column 'id_kelas' in 'field list' in C:\projectci4\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\projectci4\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `an...', 0)
#1 C:\projectci4\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `an...')
#2 C:\projectci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `an...')
#3 C:\projectci4\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `an...', Array, false)
#4 C:\projectci4\app\Models\ModelAuth.php(20): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\projectci4\app\Controllers\Auth.php(177): App\Models\ModelAuth->Daftar(Array)
#6 C:\projectci4\system\CodeIgniter.php(904): App\Controllers\Auth->Daftar()
#7 C:\projectci4\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\projectci4\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\projectci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\projectci4\system\Commands\Server\rewrite.php(43): require_once('C:\\projectci4\\p...')
#11 {main}
CRITICAL - 2023-03-24 03:20:55 --> Unknown column 'id_kelas' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (\'2\', \'623489\', \'Dinda Amelia Putri\', \'Perempuan\', \'0895613113940\', \'dinda123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (\'2\', \'623489\', \'Dinda Amelia Putri\', \'Perempuan\', \'0895613113940\', \'dinda123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (\'2\', \'623489\', \'Dinda Amelia Putri\', \'Perempuan\', \'0895613113940\', \'dinda123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (:id_kelas:, :nis:, :nama_siswa:, :jenis_kelamin:, :no_hp:, :password:)', [...], false)
 5 APPPATH\Models\ModelAuth.php(20): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Auth.php(177): App\Models\ModelAuth->Daftar([...])
 7 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Auth->Daftar()
 8 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 9 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2023-03-24 03:24:11 --> mysqli_sql_exception: Unknown column 'id_kelas' in 'field list' in C:\projectci4\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\projectci4\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `an...', 0)
#1 C:\projectci4\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `an...')
#2 C:\projectci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `an...')
#3 C:\projectci4\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `an...', Array, false)
#4 C:\projectci4\app\Models\ModelAuth.php(20): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\projectci4\app\Controllers\Auth.php(177): App\Models\ModelAuth->Daftar(Array)
#6 C:\projectci4\system\CodeIgniter.php(904): App\Controllers\Auth->Daftar()
#7 C:\projectci4\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\projectci4\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\projectci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\projectci4\system\Commands\Server\rewrite.php(43): require_once('C:\\projectci4\\p...')
#11 {main}
CRITICAL - 2023-03-24 03:24:11 --> Unknown column 'id_kelas' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (\'8\', \'620895\', \'Andy Park\', \'Laki-laki\', \'081336934682\', \'andy123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (\'8\', \'620895\', \'Andy Park\', \'Laki-laki\', \'081336934682\', \'andy123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (\'8\', \'620895\', \'Andy Park\', \'Laki-laki\', \'081336934682\', \'andy123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (:id_kelas:, :nis:, :nama_siswa:, :jenis_kelamin:, :no_hp:, :password:)', [...], false)
 5 APPPATH\Models\ModelAuth.php(20): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Auth.php(177): App\Models\ModelAuth->Daftar([...])
 7 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Auth->Daftar()
 8 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 9 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2023-03-24 03:27:08 --> mysqli_sql_exception: Unknown column 'nama_siswa' in 'field list' in C:\projectci4\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\projectci4\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `an...', 0)
#1 C:\projectci4\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `an...')
#2 C:\projectci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `an...')
#3 C:\projectci4\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `an...', Array, false)
#4 C:\projectci4\app\Models\ModelAuth.php(20): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\projectci4\app\Controllers\Auth.php(177): App\Models\ModelAuth->Daftar(Array)
#6 C:\projectci4\system\CodeIgniter.php(904): App\Controllers\Auth->Daftar()
#7 C:\projectci4\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\projectci4\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\projectci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\projectci4\system\Commands\Server\rewrite.php(43): require_once('C:\\projectci4\\p...')
#11 {main}
CRITICAL - 2023-03-24 03:27:08 --> Unknown column 'nama_siswa' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (\'8\', \'620895\', \'Andy Park\', \'Laki-laki\', \'081336934682\', \'andy123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (\'8\', \'620895\', \'Andy Park\', \'Laki-laki\', \'081336934682\', \'andy123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (\'8\', \'620895\', \'Andy Park\', \'Laki-laki\', \'081336934682\', \'andy123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (:id_kelas:, :nis:, :nama_siswa:, :jenis_kelamin:, :no_hp:, :password:)', [...], false)
 5 APPPATH\Models\ModelAuth.php(20): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Auth.php(177): App\Models\ModelAuth->Daftar([...])
 7 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Auth->Daftar()
 8 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 9 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
ERROR - 2023-03-24 03:30:08 --> mysqli_sql_exception: Unknown column 'no_hp' in 'field list' in C:\projectci4\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\projectci4\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `an...', 0)
#1 C:\projectci4\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `an...')
#2 C:\projectci4\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `an...')
#3 C:\projectci4\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `an...', Array, false)
#4 C:\projectci4\app\Models\ModelAuth.php(20): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 C:\projectci4\app\Controllers\Auth.php(177): App\Models\ModelAuth->Daftar(Array)
#6 C:\projectci4\system\CodeIgniter.php(904): App\Controllers\Auth->Daftar()
#7 C:\projectci4\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\projectci4\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\projectci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\projectci4\system\Commands\Server\rewrite.php(43): require_once('C:\\projectci4\\p...')
#11 {main}
CRITICAL - 2023-03-24 03:30:08 --> Unknown column 'no_hp' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (\'8\', \'620895\', \'Andy Park\', \'Laki-laki\', \'081336934682\', \'andy123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (\'8\', \'620895\', \'Andy Park\', \'Laki-laki\', \'081336934682\', \'andy123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (\'8\', \'620895\', \'Andy Park\', \'Laki-laki\', \'081336934682\', \'andy123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `anggota` (`id_kelas`, `nis`, `nama_siswa`, `jenis_kelamin`, `no_hp`, `password`) VALUES (:id_kelas:, :nis:, :nama_siswa:, :jenis_kelamin:, :no_hp:, :password:)', [...], false)
 5 APPPATH\Models\ModelAuth.php(20): CodeIgniter\Database\BaseBuilder->insert([...])
 6 APPPATH\Controllers\Auth.php(177): App\Models\ModelAuth->Daftar([...])
 7 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Auth->Daftar()
 8 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 9 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
CRITICAL - 2023-03-24 03:44:03 --> Undefined index: nama_rak
in APPPATH\Views\v_rak.php on line 39.
 1 APPPATH\Views\v_rak.php(39): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index: nama_rak', 'APPPATH\\Views\\v_rak.php', 39, [...])
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_rak.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_rak', [], true)
 5 APPPATH\Views\v_template_admin.php(249): view('v_rak')
 6 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\v_template_admin.php')
 7 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1137): CodeIgniter\View\View->render('v_template_admin', [], true)
 9 APPPATH\Controllers\Rak.php(27): view('v_template_admin', [...])
10 SYSTEMPATH\CodeIgniter.php(904): App\Controllers\Rak->index()
11 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Rak))
12 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(43): require_once('FCPATH\\index.php')
